### PR TITLE
fix: versioning + ci python version fix

### PR DESCRIPTION
**Pull request by riomus**

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -18,36 +18,40 @@ packages = [
...
 datasets = "1.6.1"
-Pillow = "^8.2.0"
-tensorboard = "^2.4.1"
-typer = "^0.3.2"
-spacy = "^3.0.0"
-seqeval = "^1.2.2"
+Pillow = "8.2.0"
+tensorboard = "2.4.1"
+typer = "0.3.2"
+spacy = "3.0.0"
```
Comment (albert097) [inline pyproject.toml:32]:
```suggestion
spacy = "3.1.0"
```

File: .github/workflows/python_poetry_main.yml [hunk 2]
```diff
@@ -28,9 +28,9 @@ jobs:
         run: poetry run poe test
   lint:
     strategy:
-      fail-fast: true
+      fail-fast: false
       matrix:
-        python-version: [3.8]
+        python-version: [3.9.6]
         poetry-version: [1.1.5]
```
Comment (albert097) [inline .github/workflows/python_poetry_main.yml:34]: Whole file should be formatted ctrl+alt+l

File: .gitignore [hunk 3]
```diff
@@ -134,4 +134,7 @@ dmypy.json
 resources/
 
 # compiled site
-webpage/public
\ No newline at end of file
+webpage/public
+
+#excluding pyenv
+!.python-version
```
Comment (albert097) [inline .gitignore:140]:
```suggestion
!.python-version

```

add new line

**Review by albert097 — CHANGES_REQUESTED** [review]

File: pyproject.toml [hunk 4]
```diff
@@ -18,36 +18,40 @@ packages = [
...
 black = "black --check ./"
 isort = "isort .  -c"
+black_fix = "black ./"
+isort_fix  = "isort . "
 pyflakes = "pyflakes ."
 mypy  = "mypy --strict ."
 test = "pytest"
 check       =  ["black", "isort", "mypy",  "pyflakes"]
+fix       =  ["black_fix", "isort_fix"]
 all         = ["check", "test"]
```
Comment (albert097) [inline pyproject.toml:55]: should be formatted ctrl+alt+l

**Review by albert097 — CHANGES_REQUESTED** [review]

**Review by albert097 — DISMISSED** [review]

**Review by ktagowski — APPROVED** [review]

**Review by albert097 — APPROVED** [review]

**Pull request merged**